R/staticimports.R

```diff
@@ -81,3 +81,35 @@ str_extract <- function(text, pattern) {
     )
   )
 }
+# Generated by staticimports; do not edit by hand.
+# ======================================================================
+# Imported from pkg:stringstatic
+# ======================================================================
+
+`%||%` <- function(lhs, rhs) {
+  if (length(lhs) == 0) return(rhs)
+  lhs
+}
+
+fixed <- function(pattern, ignore_case = FALSE) {
+  structure(
+    pattern,
+    options = list(case_insensitive = ignore_case),
+    class = c("fixed", "pattern", "character")
+  )
+}
+
+str_count <- function(string, pattern) {
+  is_fixed <- inherits(pattern, "fixed")
+  ignore.case <- attr(pattern, "options")$case_insensitive %||% FALSE
```

Comment:
I'd recommend you rewrite the static export to avoid `%||%`, it'd be more portable that way and only a little more verbose.

Reply:
Good thinking! All it took was an `isTRUE()`, which is much more elegant if I do say so myself [281f2ae](https://github.com/rstudio/gradethis/pull/325/commits/281f2ae708e1ddf20b6f7d4f2b1ce6ab5e0d87c7)